### DISCUSSION TITLE
contexual orders in subeditor prevents interacting with items in containers

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#15155] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

used contexual orders in sub editor > got unavailability to drop, drag, delete items in containers
bug persist after reentering sub editor from menu and probably on whole session.
![изображение](https://github.com/user-attachments/assets/dd1cd4d5-5b6a-4441-84d7-4440dc763fe4)
*transparent ui mod made by me for me

### Reproduction steps

subeditor>go into test mode and go back to editor>open container>shift+mmb any item in container>observe the bug

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
no
```

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately couldn't get the issue to happen at my end, clicking shift+MMB on the item seems to do nothing nor affect moving/dropping/placing items. I wonder if the mod you're using could have something to do with the issue? What exactly happens when you click MMB, I'm assuming it doesn't open the contextual menu but does anything else happen?

**Reply by DrBruhman** [reply]
i reproduced the bug => it happens after exiting testing mode and not happens when it fresh game session

**Reply by DrBruhman** [reply]
also it fixes itself when i reenter testing mode

**Comment by Regalis11** [comment]
Thank you for the report! This has been addressed in the current unstable build, fix coming up to the public version in a future update.
